### ISSUE TITLE
Collision with default Vagrant IPv4 addresses yields minimally helpful error msg

### ISSUE THREAD

**Issue by syreal17** [picoCTF/picoCTF#291]
TL;DR 1: Let's choose a better default IPv4 addr for "web" and "shell". 2: Vagrant needs to fail more helpfully if there is a network collision (please no halfways VM's lingering which are misleading)

Workaround: Thankfully, previous picoCTF contributors have provided usage of "WIP" (Web IP) and "SIP" (Shell IP) environment variables to deployers as an undocumented way to override default 192.168.2.2 and 192.168.2.3 IPv4 addresses respectively. However, this seems to be unconnected to nginx configuration as the web server did not successfully serve web pages on "WIP" after "vagrant up"

Narrative:
I had a lot of problems deploying picoCTF framework on my personal Alienware laptop. At first, I thought it was Windows related, but I discovered in combing through Vagrant "up" output that it was a network conflict:

```
==> shell: Checking if box 'ubuntu/bionic64' version '20190429.0.0' is up to date...
==> shell: Setting the name of the VM: picoCTF-shell-dev
Vagrant is currently configured to create VirtualBox synced folders with
the `SharedFoldersEnableSymlinksCreate` option enabled. If the Vagrant
guest is not trusted, you may want to disable this option. For more
information on this option, please refer to the VirtualBox manual:

  https://www.virtualbox.org/manual/ch04.html#sharedfolders

This option can be disabled globally with an environment variable:

  VAGRANT_DISABLE_VBOXSYMLINKCREATE=1

or on a per folder basis within the Vagrantfile:

  config.vm.synced_folder '/host/path', '/guest/path', SharedFoldersEnableSymlinksCreate: false
==> shell: Clearing any previously set network interfaces...
The specified host network collides with a non-hostonly network!
This will cause your specified IP to be inaccessible. Please change
the IP or name of your host only network so that it no longer matches that of
a bridged or non-hostonly network.

Bridged Network Address: '192.168.2.0'
Host-only Network 'Killer Wireless-n/a/ac 1435 Wireless Network Adapter': '192.168.2.0'

C:\Users\maran\github\syreal17\picoCTF>
```

Not a terrible error message, though a touch subtle. What's most unhelpful is that the "shell" machine half-succeeds in provisioning and "web" machine completely aborts. This is a particularly difficult way to learn Vagrant if this is the first time a deployer is using Vagrant. "192.168.2.0/24" is the default host-only network that both "shell" and "web" are deployed to, which seems like a bad choice, as seen on my personal, this is a common network for a default home WiFi router.
